### PR TITLE
[REQUIRES #4962] Remove global collision and update examples

### PR DESCRIPTION
**Pull request by howard-stearns**
This is the second part of https://app.asana.com/0/32622044445063/34250577852897 "remove old 'global' collision callback from JS".

It is a continuation of, and should only be merged after, https://github.com/highfidelity/hifi/pull/4962 "Entity events in normal scripts".

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d214f128cd5061c4c70a26b57b89f50501faaa09 available at: https://gist.github.com/b19db72122685501361e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.dmg

File: examples/example/entityCollisionExample.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+//
+//  globalCollisionsExample.js
+//  examples
+//
+//  Created by Brad Hefta-Gaub on 1/29/14.
```
Comment (ZappoMan) [inline examples/example/entityCollisionExample.js]: update header

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae73e68f7bdf0282bbf2eda322da8492efb25942 available at: https://gist.github.com/b19db72122685501361e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.dmg

**Comment by ZappoMan** [comment]
@howard-stearns - can you merge with upstream so as to make this diff easier to read

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4b4bf9fd90a419054bcf3c5cd2ad76de390fdddd available at: https://gist.github.com/b19db72122685501361e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4964.dmg

**Pull request merged**